roles/manage_cnp/handlers/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# handlers file for manage_cnp
```

Comment:
remove this file and the handlers directory for this role

Reply:
removed